tests/cases/probe_without_clear_command.expect_fail.amber

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[vertex shader passthrough]
+
+[fragment shader]
+#version 430
+
+layout(location = 0) in vec4 color_in;
+layout(location = 0) out vec4 color_out;
+
+void
+main()
+{
+  color_out = color_in;
+}
+
+[test]
+relative probe rect rgba (0.0, 0.0, 1.0, 1.0) (0.1, 0.2, 0.3, 0.4)
```

Comment:
So, this is going to be potentially flaky won't it? We have no idea what's going to be in the buffer at that position, so we could possibly match this probe by accident?

Should we set the color_out to be vec4(.5, .5, .5, .5) so it's a known value?

Reply:
Can we set epsilon to infinity (or close enough), and then probe a nonsensical value and it should always pass.

Reply:
The issue here is that we should accept the probe without any operation.
I discuss it with @dj2 in offline and we should print an error message if it probes without any operations i.e., no draw, no clear because it is not meaningful and it would be probably a mistake from the script writer.

Let me just keep the assert I removed. This test should fail without the assert. I will put a comment for the failure message at the bottom of this script.